src/Facade/Services/StockTriggerLevelsFacadeService.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Linn.Stores.Facade.Services
+{
+    using System;
+    using System.Linq.Expressions;
+
+    using Linn.Common.Facade;
+    using Linn.Common.Persistence;
+
+    using Linn.Stores.Domain.LinnApps.StockLocators;
+
+    using Linn.Stores.Resources;
+
+    public class StockTriggerLevelsFacadeService : FacadeService<StockTriggerLevel, int, StockTriggerLevelsResource, StockTriggerLevelsResource>, IStockTriggerLevelsFacadeService
+    {
+        private readonly IRepository<StockTriggerLevel, int> repository;
+
+        private readonly ITransactionManager transactionManager;
+
+        public StockTriggerLevelsFacadeService(IRepository<StockTriggerLevel, int> repository, ITransactionManager transactionManager) : base(repository, transactionManager)
+        {
+            this.repository = repository;
+            this.transactionManager = transactionManager;
+        }
+
+        public IResult<StockTriggerLevel> DeleteStockTriggerLevel(int id)
+        {
+            var toDelete = this.repository.FindBy(s => s.Id == id);
```

Comment:
i would at least log that this has happened. maybe capture the user number too. 
I was going to say that you don't need the transaction manager variable above but you probably need it here as I don't think this is being committed in the module either